package.json

```diff
@@ -193,454 +193,18 @@
-    },
-    "./editors/passwordEditor": {
-      "import": "./editors/passwordEditor/index.mjs",
-      "require": "./editors/passwordEditor/index.js"
-    },
-    "./editors/selectEditor": {
-      "import": "./editors/selectEditor/index.mjs",
-      "require": "./editors/selectEditor/index.js"
-    },
-    "./editors/textEditor": {
-      "import": "./editors/textEditor/index.mjs",
-      "require": "./editors/textEditor/index.js"
-    },
-    "./i18n": {
-      "import": "./i18n/index.mjs",
-      "require": "./i18n/index.js"
-    },
-    "./i18n/languages": {
-      "import": "./i18n/languages/index.mjs",
-      "require": "./i18n/languages/index.js"
-    },
-    "./i18n/languages/de-CH": {
-      "import": "./i18n/languages/de-CH.mjs",
-      "require": "./i18n/languages/de-CH.js"
-    },
-    "./i18n/languages/de-DE": {
-      "import": "./i18n/languages/de-DE.mjs",
-      "require": "./i18n/languages/de-DE.js"
-    },
-    "./i18n/languages/en-US": {
-      "import": "./i18n/languages/en-US.mjs",
-      "require": "./i18n/languages/en-US.js"
-    },
-    "./i18n/languages/es-MX": {
-      "import": "./i18n/languages/es-MX.mjs",
-      "require": "./i18n/languages/es-MX.js"
-    },
-    "./i18n/languages/fr-FR": {
-      "import": "./i18n/languages/fr-FR.mjs",
-      "require": "./i18n/languages/fr-FR.js"
-    },
-    "./i18n/languages/it-IT": {
-      "import": "./i18n/languages/it-IT.mjs",
-      "require": "./i18n/languages/it-IT.js"
-    },
-    "./i18n/languages/ja-JP": {
-      "import": "./i18n/languages/ja-JP.mjs",
-      "require": "./i18n/languages/ja-JP.js"
-    },
-    "./i18n/languages/ko-KR": {
-      "import": "./i18n/languages/ko-KR.mjs",
-      "require": "./i18n/languages/ko-KR.js"
-    },
-    "./i18n/languages/lv-LV": {
-      "import": "./i18n/languages/lv-LV.mjs",
-      "require": "./i18n/languages/lv-LV.js"
-    },
-    "./i18n/languages/nb-NO": {
-      "import": "./i18n/languages/nb-NO.mjs",
-      "require": "./i18n/languages/nb-NO.js"
-    },
-    "./i18n/languages/nl-NL": {
-      "import": "./i18n/languages/nl-NL.mjs",
-      "require": "./i18n/languages/nl-NL.js"
-    },
-    "./i18n/languages/pl-PL": {
-      "import": "./i18n/languages/pl-PL.mjs",
-      "require": "./i18n/languages/pl-PL.js"
-    },
-    "./i18n/languages/pt-BR": {
-      "import": "./i18n/languages/pt-BR.mjs",
-      "require": "./i18n/languages/pt-BR.js"
-    },
-    "./i18n/languages/ru-RU": {
-      "import": "./i18n/languages/ru-RU.mjs",
-      "require": "./i18n/languages/ru-RU.js"
-    },
-    "./i18n/languages/zh-CN": {
-      "import": "./i18n/languages/zh-CN.mjs",
-      "require": "./i18n/languages/zh-CN.js"
-    },
-    "./i18n/languages/zh-TW": {
-      "import": "./i18n/languages/zh-TW.mjs",
-      "require": "./i18n/languages/zh-TW.js"
-    },
-    "./languages": {
-      "import": "./languages/index.mjs",
-      "require": "./languages/index.js"
-    },
+    "./cellTypes/*": {},
```

Comment:
Now when the `package.json` file is autogenerated by the script. It can be confusing to keep the `"exports"` key in the original file.

Someone familiar with the specs about [Modules](https://nodejs.org/api/packages.html#packages_subpath_patterns) can be surprised with that kind of entry `"./cellTypes/*": {},` - it does not meet the specification. I think that it'd be a better idea to rename that `"export"` key to something customized. Maybe we can use the `"handsontable"` (or `"hot"`) key?

Something like:
```json
"handsontable": {
  "exports": {
    ".": {
      "import": "./index.mjs",
      "require": "./index.js"
    },
    "./base": {
      "import": "./base.mjs",
      "require": "./base.js"
    },
    "./cellTypes/*": {},
    "./editors/*": {},
    "./i18n/*": {},
    "./i18n/languages/*": {},
    "./languages/*": {},
    "./languages/all": {
      "import": "./languages/index.mjs",
      "require": "./languages/all.js"
    },
    "./plugins/*": {},
    "./renderers/*": {},
    "./validators/*": {},
    "./dist/handsontable.css": "./dist/handsontable.css",
    "./dist/handsontable.full.css": "./dist/handsontable.full.css",
    "./dist/handsontable.full.min.css": "./dist/handsontable.full.min.css",
    "./dist/handsontable.min.css": "./dist/handsontable.min.css"
  }
}
```

WDYT?

Reply:
Maybe we could use `glob` package (it's already added in our dev dependencies). So it'd fix the side-effect mentioned [here](https://github.com/handsontable/handsontable/pull/7828#pullrequestreview-640862192). For example for the `"./i18n/languages/*": {},` entry we could rewrite to something like this `"./i18n/languages/*": "./src/i18n/+(languages|registry)*",` (or as an array of glob patterns)?

Reply:
Thank you @budnix for your great feedback. I adjusted the code following your suggestions.
I also added some other things we discussed internally on Slack.